feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -291,3 +291,8 @@ private fun CameraUseCase.AspectRatio.toUiState() = when(this) {
     CameraUseCase.AspectRatio.ASPECT_RATIO_16_9 -> AspectRatio.NINE_SIXTEEN
     CameraUseCase.AspectRatio.ASPECT_RATIO_1_1 -> AspectRatio.ONE_ONE
 }
+
+private fun CameraUseCase.CaptureMode.toUiState() = when(this) {
+    CameraUseCase.CaptureMode.MULTI_STREAM -> CaptureMode.MULTI_STREAM
```

Comment:
Same comment as the last PR; it feels like it only makes sense to add new enums to the UI state if they are fundamentally different from the enums they're derived from. Otherwise they could just be part of the model. Probably OK for now, but we might want to consolidate in another PR.